csm_web/frontend/static/frontend/css/style.css

```diff
@@ -82,6 +84,19 @@ header {
 	color: #808080;
 }
 
+.site-subtitle-link {
+	color: #bbbbbb;
+	text-decoration: none;
+	margin-top: auto;
+	margin-bottom: auto;
+	margin-left: auto;
+	margin-right: 0px;
```

Comment:
You can combine these into one `margin` style.

Reply:
updated to 1 line